data/data.go

```diff
@@ -31,34 +31,39 @@ type EffxYaml struct {
 	FilePath string
 }
 
-// prevents us from overwriting tags and annotations that the customer
```

Comment:
I'm removing this since the CLI is responsible for adding tags and annotations to the `ConfigurationFile`